### DISCUSSION TITLE
Reliability and process for determining and releasing fixes

### DISCUSSION THREAD

**Discussion by duncanhall** [bbc/sqs-consumer#353] [Help] [answered]
This library is an excellent contribution and solution to dealing with the various parts of consuming SQS queues. However there seem to be quite a few fundamental issues being reported which make it hard to select this library for production use. 

- As this library is published under the BBC organisation, and not published under an individual contributor - what level of support/time/resource is allocated by the BBC organisation for this library and it's ongoing maintenance?
- How can we create a process which makes it easier for us to respond to 'help wanted' labels?

**Comment by nicholasgriffintn** [comment, marked as answer]
Hey, some answers as close as I can answer:

1: This library is under maintenance and within an as needed schedule, this means there's no defined timelines, as a relatively small library, there's not much justification for more than that.

2: We are open for contributions from anyone who wants to contribute, according to our [Contributors Guidelines](https://github.com/bbc/sqs-consumer/blob/main/.github/CONTRIBUTING.md).
